src/KernelAbstractions.jl

```diff
@@ -1,14 +1,19 @@
 module KernelAbstractions
 
 export @kernel
-export @Const, @localmem, @private, @uniform, @synchronize, @index, groupsize
+export @Const, @localmem, @private, @uniform, @synchronize, @index, groupsize, pin!, recordevent, async_copy!
 export Device, GPU, CPU, CUDA 
 
 using MacroTools
 using StaticArrays
 using Cassette
 using Requires
 using Adapt
+using CUDAdrv
```

Comment:
Any code dependent on CUDAdrv being available needs to move into `backends/cuda.jl`